prowler/lib/check/checks_loader.py

```diff
@@ -7,15 +7,15 @@
 
 # Generate the list of checks to execute
 def load_checks_to_execute(
-    bulk_checks_metadata: dict,
-    bulk_compliance_frameworks: dict,
-    checks_file: str,
-    check_list: list,
-    service_list: list,
-    severities: list,
-    compliance_frameworks: list,
-    categories: set,
-    provider: str,
+    bulk_checks_metadata: dict = None,
+    bulk_compliance_frameworks: dict = None,
```

Comment:
I think these should be always required, what do you think?

Reply:
I've made both optionals, the only one that it's really needed is the `bulk_checks_metadata`, that will fill all the dicts from `check_aliases`, `check_categories`, `check_severities` I've tried to get the bulk checks metadata only if a category or severity it's passed but the main problem resides on the alias from the check. We always update the checks with them aliases so check metadata it's needed.

Reply:
Thanks for the explanation!